sympy/core/power.py

```diff
@@ -530,6 +530,9 @@ def _eval_is_extended_positive(self):
                 return log(self.base).is_imaginary
 
     def _eval_is_extended_negative(self):
+        if self.exp is S(1)/2:
+            if self.base.is_complex or self.base.is_extended_real:
+                return False
```

Comment:
I thought the way these assumptions worked was that they should return what is True and the logic handling would work out what is None or False.  Whether or not that changes anything here, I think we need `if self.base.is_real: return False` added.

Reply:
Since `real` implies `extended_real` we have:
```
In [6]: x = Symbol('x', real=True)                                                                                                             

In [7]: x.is_extended_real                                                                                                                     
Out[7]: True
```
The assumptions system works that out internally so anything that would pass `if self.base.is_real` will also pass `if self.base.is_extended_real`.

So when writing these checks we can always check for the most general predicate possible. At the moment complex and extended_real are the most general number predicates.